pkg/kyma/cmd/test/delete/cmd.go

```diff
@@ -0,0 +1,68 @@
+package del
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test/client"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "delete",
+		Short:   "Delete tests on a running Kyma cluster",
+		Long:    `Delete tests on a running Kyma cluster`,
+		RunE:    func(_ *cobra.Command, args []string) error { return cmd.Run(args) },
+		Aliases: []string{"d"},
+	}
+
+	cobraCmd.Flags().BoolVarP(&o.All, "all", "a", false, "Delete all test suites")
+	return cobraCmd
+}
+
+func (cmd *command) Run(args []string) error {
+	if len(args) < 1 && !cmd.opts.All {
+		return fmt.Errorf("test suite name requied")
+	}
+
+	cli, err := client.NewTestRESTClient(10 * time.Second)
+	if err != nil {
+		return fmt.Errorf("unable to create test REST client. E: %s", err)
+	}
+
+	if cmd.opts.All {
```

Comment:
So if you specify an arg but no -a, then nothing will happen?

Reply:
If you specify a test suite name as the first argument, it will remove only this test suite. It's possible to provide multiple test suite names, ex: `kyma test delete test-1080664667 test-108061239`